sdk/src/DocuSign.eSign/DocuSign.eSign.nuspec

```diff
@@ -1,41 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
@RajRele is this intended? How will Nuget release work?

Reply:
@mmallis87 Yes we moved to sdk-style project structure, we don't need to have nuspec files anymore.
https://docs.microsoft.com/en-us/nuget/reference/nuspec#project-type-compatibility